modules/rosa-sdpolicy-am-compute.adoc

```diff
@@ -14,7 +14,7 @@ All {product-title} clusters support a maximum of 180 worker nodes.
 
 [NOTE]
 ====
-The `Default` machine pool node type and size cannot be changed after the cluster is created.
+The `Default` machine pool node type and size cannot be changed after the cluster is created. To change the `Default` machine pool node type and size, delete the current `Default` machine pool and then create a new `Default` machine pool with the desired settings. For more information regarding machine pools, see xref:../../rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc#rosa-managing-worker-nodes[Managing compute nodes].
```

Comment:
I prefer enhance the doc for include the default machinepool's id/name included in the command after the feature enabled

Reply:
The term "default" machine pool will no longer be used based on information provided from the architect that is associated with the JIRA ticket this PR is tied to. Therefore the comment has been removed entirely until further clarification is presented.